lib/tasks/github_usage_tracker.rake

```diff
@@ -0,0 +1,6 @@
+namespace :github_usage_tracker do
+  desc "Record a current datapoint of GitHub API usage"
+  task :record_datapoint => :environment do
+    GithubUsageTracker.record_datapoint
+  end
```

Comment:
This will be run on a cron. Can add more details about setting up InfluxDB and crons in another PR.